src/pretix/control/context.py

```diff
@@ -26,10 +26,14 @@ def contextprocessor(request):
             _html_head.append(response)
     ctx['html_head'] = "".join(_html_head)
 
+    _js_datetime_weekdays = '[]'
     _nav_event = []
     if hasattr(request, 'event'):
         for receiver, response in nav_event.send(request.event, request=request):
             _nav_event += response
+        if request.event.settings.get('payment_term_weekdays'):
+            _js_datetime_weekdays = '[0,6]'
```

Comment:
When I first read this, I was very confused -- is this tha days that are disabled? Or the enabled ones? How about naming the variable (and the HTML attribute) something like ``payment_weekdays_disabled`` instead of ``datetimes_weekdays``?